Sender: LSF System <lsfadmin@xc02n15>
Subject: Job 477841: <CH4-HF-B3> in cluster <xdft1> Exited

Job <CH4-HF-B3> was submitted from host <xcluster> by user <zyzhu> in cluster <xdft1> at Fri Mar 29 10:48:16 2019
Job was executed on host(s) <24*xc02n15>, in queue <xp24mc2>, as user <zyzhu> in cluster <xdft1> at Fri Mar 29 11:03:48 2019
</share/home/zyzhu> was used as the home directory.
</share/home/zyzhu/Documents-Shared/HF_DFT_related/src/examples/CH4> was used as the working directory.
Started at Fri Mar 29 11:03:48 2019
Terminated at Fri Mar 29 11:04:27 2019
Results reported at Fri Mar 29 11:04:27 2019

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
#BSUB -n 24
#BSUB -q xp24mc2
#BSUB -J CH4-HF-B3
#BSUB -R span[ptile=24]
#BSUB -o CH4-%J.out

python CH4.py > CH4.log

------------------------------------------------------------

Exited with exit code 1.

Resource usage summary:

    CPU time :                                   436.00 sec.
    Max Memory :                                 32152 MB
    Average Memory :                             6019.14 MB
    Total Requested Memory :                     -
    Delta Memory :                               -
    Max Swap :                                   -
    Max Processes :                              4
    Max Threads :                                51
    Run time :                                   39 sec.
    Turnaround time :                            971 sec.

The output (if any) follows:

Traceback (most recent call last):
  File "CH4.py", line 38, in <module>
    scfh = HFHelper(mol)
  File "../../hf_helper.py", line 75, in __init__
    self.initialization_scf()
  File "../../hf_helper.py", line 111, in initialization_scf
    self.eri0_mo = np.einsum("uvkl, up, vq, kr, ls -> pqrs", self.eri0_ao, self.C, self.C, self.C, self.C)
  File "/share/home/zyzhu/anaconda3/lib/python3.6/site-packages/numpy/core/einsumfunc.py", line 1297, in einsum
    new_view = c_einsum(einsum_str, *tmp_operands, **einsum_kwargs)
ValueError: iterator is too large
